src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -527,20 +529,20 @@ Execute the Workflow and set the different workflow's variables as follows:
 | *Variable name* | *Description* | *Type*
 3+^|*Workflow variables*
 | `SERVICE_TOKEN`
-| A valid token required to be able to deploy the model.
+| A valid token generated by the MaaS Service for user authentication.
 | String
 | `DOCKER_ENABLED`
 |   If True, the workflow will be executed on a Docker container. link:https://hub.docker.com/r/activeeon/dlm3/[activeeon/dlm3] image is used by default. If False, the required libraries should be installed in the different nodes that will be used.
 | Boolean (default=True)
 3+^|*Task variables*
 | `DEPLOY_MODEL_ENDPOINT`
-| The endpoint of the started service.
+| A URL endpoint defined by the user where the AI Model was deployed
```

Comment:
An instead of A ?

Reply:
where the AI Model was deployed => referencing the AI Model which was deployed
If u are not ok with this suggestion, keep it like it is :))

Reply:
It is more correct to use `a` here.
"Although the U at the first of URL is a vowel, the abbreviation is pronounced “you-are-ell.” Because the letter Y is a consonant in this case, use a rather than an."